### DISCUSSION TITLE
URL not found when behind Proxy with Apache (httpd) using subpaths.

### DISCUSSION THREAD

**Discussion by llorencs** [fastapi/fastapi#8833] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# v1.py
from fastapi import fastAPI
app_v1 = FastApi(root_path='/api/v1')

@app_v1.get("/")
async def hello():
    return{"Hello"}

# main.py
from fastapi import fastAPI
from v1 import app_v1
import uvicorn

app = FastAPI(root_path('/api/v1')
app.mount('/api/v1', app_v1)

if __name__== '__main__':
   uvicorn.run(app,host='0.0.0.0', port=8001, debug=True)
```

### Description

The uvicorn server is behind an Apache proxy with this configuration.

```
<IfModule mod_proxy.c>
        ProxyRequests Off
        ProxyPreserveHost On
        ProxyPass /api http://localhost:8001/ Keepalive=On
        ProxyPassReverse /api http://localhost:8001/
        ProxyTimeout 3600
</IfModule>
```
When accessing:
https://localhost/api/v1
is expected to return {'Hello'}

But instead, it returns
{"detail": "Not Found"}

As the URL has not been found. What am I doing wrong?

### Operating System

Linux

### FastAPI Version

0.68.0

### Python Version

3.8.6

**Comment by STeveShary** [comment, marked as answer]
@llorencs This is a mis-configuration of your apache configuration, not a FastAPI issue. I believe the following will work:
```
<IfModule mod_proxy.c>
        ProxyRequests Off
        ProxyPreserveHost On
        ProxyPass /api http://localhost:8001/api Keepalive=On
        ProxyPassReverse /api http://localhost:8001/api
        ProxyTimeout 3600
</IfModule>
```
